internal/tfengine/tfengine.go

```diff
@@ -225,6 +229,20 @@ func dumpTemplate(conf *Config, pwd, cacheDir, outputPath string, ti *templateIn
 	return nil
 }
 
+func addGitIgnore(path string) error {
```

Comment:
Can we make this a component and call it through something like devops recipe?

Reply:
I was initially thinking of doing that but felt that it doesn't really belong to any recipe or component. On a second thought, i think it doesn't make sense to not having any TF configs and still want to ignore tf temporary files. And devops is the most foundational block. I'll add it to the  devops recipe

Reply:
Done